sensing_launch/launch/velodyne_VLP16.launch.xml

```diff
@@ -4,101 +4,19 @@
   <arg name="launch_driver" default="true" />
 
   <arg name="model" default="VLP16"/>
-  <arg name="calibration" default="$(find velodyne_pointcloud)/params/VLP16db.yaml"/>
-  <arg name="device_ip" default="192.168.1.201" />
-  <arg name="sensor_frame" default="velodyne" />
-  <arg name="base_frame" default="base_link" />
-  <arg name="manager" default="velodyne_nodelet_manager" />
+  <arg name="calibration" default="$(find-pkg-share velodyne_pointcloud)/params/VLP16db.yaml"/>
   <arg name="max_range" default="130.0" />
   <arg name="min_range" default="0.4" />
-  <arg name="pcap" default="" />
-  <arg name="port" default="2368" />
-  <arg name="read_fast" default="false" />
-  <arg name="read_once" default="false" />
-  <arg name="repeat_delay" default="0.0" />
-  <arg name="rpm" default="600.0" />
-  <arg name="scan_phase" default="0.0" />
-  <arg name="sensor_timestamp" default="false" />
-  <arg name="laserscan_ring" default="-1" />
-  <arg name="laserscan_resolution" default="0.007" />
-  <arg name="num_points_threshold" default="300"/>
   <arg name="invalid_intensity" default="[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]"/>
 
```

Comment:
The velodyne_VLP16.launch.xml should have `sensor_frame`, `device_ip`, `port` as input arguments since they are passed from lidar.launch.xml.
Same goes for velodyne_VLP32C.launch.xml and velodyne_VLS128.launch.xml

```suggestion
  <arg name="sensor_frame" default="velodyne" />
  <arg name="device_ip" default="192.168.1.201"/>
  <arg name="port" default="2368"/>
```

Reply:
done